lib/splash.js

```diff
@@ -39,7 +41,13 @@ function logKey(key) {
 }
 
 function logValue(key, value) {
-    return "| " + indent() + " " + key + ": " + value + "\n";
+    var hideValue = _.reduce(KEYS_TO_HIDE, function(memo, keyToHide) {
+        return memo || key.indexOf(keyToHide) >= 0;
```

Comment:
Oh okay ! We do not have such properties in av-accounts.
I'll update it.
Add a logArray function seems to be a good solution